blockmanager.go

```diff
@@ -1273,9 +1273,13 @@ func (b *blockManager) resolveConflict(
 					log.Errorf("Unable to ban peer %v: %v",
 						peer, err)
 				}
+				if sp := b.server.PeerByAddr(peer); sp != nil {
```

Comment:
Could've sworn that `BanPeer` _did_ disconnect the peer, but this isn't the case as is...

Reply:
> Could modify the method to do so as well

+1

Reply:
Done.